packages/util/src/__specs__/class-names.spec.ts

```diff
@@ -0,0 +1,11 @@
+test('classNames()', () => {
+  expect(classNames('class-1', 'class-2')).toBe('class-1 class-2')
+  const condtionTrue = true
+  const conditionFalse = false
+  const actual = classNames(
+    'class-1',
+    conditionFalse && 'class-2',
+    condtionTrue && 'class-3'
+  )
+  expect(actual).toBe('class-1 class-3')
+})
```

Comment:
Opinion: Splitting a spec into different tests, with (commonly) one assertion each can be good for lots of things, including tdd flow, easier debugs on tests failures, better documentation in spec as to what is being asserted.

Reply:
Sure I copied some old code and a test I wrote 3 or so years ago over for this. I can clean it up.